dev/tests/api-functional/testsuite/Magento/GraphQl/Customer/UpdateCustomerTest.php

```diff
@@ -42,6 +42,51 @@ protected function setUp()
         $this->lockCustomer = Bootstrap::getObjectManager()->get(LockCustomer::class);
     }
 
+    /**
```

Comment:
Need to test also scenarios where `currentPassword` is not set and exception is thrown.

Reply:
Resolved by d3e5082 commit